packages/auth/src/ActivateAccount/ActivateAccountForm.tsx

```diff
@@ -64,15 +64,18 @@ export const ActivateAccountForm: FC<ActivateAccountFormProps> = (props) => {
           name='password'
           label={t('auth.activate-account.new-password')}
           placeholder={t('auth.activate-account.enter-your-password')}
+          data-test-id='password-box'
         />
         <FInput
           type='password'
           name='confirmPassword'
           label={t('auth.activate-account.confirm-new-password')}
           placeholder={t('auth.activate-account.enter-your-password-again')}
+          data-test-id='confirmPassword-box'
```

Comment:
please use the same naming convention, choose camelCase or dash-case  do not combine between them.
confirmPassword-box should be confirm-password-box or confirmPasswordBox